account_financial_report/report/templates/trial_balance.xml

```diff
@@ -34,50 +36,65 @@
                     <t t-call="account_financial_report.report_trial_balance_lines_header"/>
 
                     <!-- Display each lines -->
-                    <t t-foreach="o.account_ids.filtered(lambda a: not a.hide_line)" t-as="line">
-                        <t t-set="type" t-value='"account_type"'/>
+                    <t t-foreach="trial_balance" t-as="balance">
                         <!-- Adapt -->
-                        <t t-set="style" t-value="'font-size:8px;'"/>
-                        <t t-set="padding" t-value="line.level * 4"/>
-                        <t t-if="o.hide_account_at_0">
-                            <t t-set="style" t-value="'font-size: 14px;'"/>
+                        <t t-set="style" t-value="'font-size:12px;'"/>
+                        <t t-if="hierarchy_on == 'none'">
+                            <t t-call="account_financial_report.report_trial_balance_line"/>
                         </t>
-                        <t t-if="o.hierarchy_on != 'none'">
-                            <t t-set="style" t-value="'font-size: ' + str(14 - line.level) + 'px; margin-left: ' + str(line.level * 4) + 'px;'"/>
+                        <t t-if="hierarchy_on == 'computed'">
+                            <t t-if="balance['type'] == 'group_type'">
+                                <t t-set="style" t-value="style + 'font-weight: bold; color: blue;'"/>
+                                    <t t-call="account_financial_report.report_trial_balance_line"/>
+                            </t>
+                            <t t-if="balance['type'] == 'account_type'">
+                                <t t-call="account_financial_report.report_trial_balance_line"/>
+                            </t>
                         </t>
-                        <t t-if="line.account_group_id">
-                            <t t-set="style" t-value="style + 'font-weight: bold; color: blue;'"/>
+<!--                        <t t-set="padding" t-value="line.level * 4"/>-->
+<!--                        <t t-if="hierarchy_on != 'none'">-->
+<!--                            <t t-set="style" t-value="'font-size: ' + str(14 - line.level) + 'px; margin-left: ' + str(line.level * 4) + 'px;'"/>-->
+<!--                        </t>-->
```

Comment:
Lot's of commented out XML code like this. And because v13 migration was based on this PR, v13 is now kind of messy.
The code commented here is also one of the better UX features of this module.
It makes the Trial Balance look like this (before PR):
![image](https://user-images.githubusercontent.com/17097810/79563934-61b6e200-80b6-11ea-8745-e90ed8de69e5.png)
Instead of like this (taken from v13):
![image](https://user-images.githubusercontent.com/17097810/79564018-8dd26300-80b6-11ea-9e3a-8af6ca10207f.png)

Also in v13 clicking on different parts of the table (like group code, account name or even the sums of money) doesn't open related data. This is also one of the best "selling points" of the module.
Though, I haven't checked if this is the case with this PR.